README.md

```diff
@@ -142,12 +137,26 @@ The following methods are exposed on an instantiated client
-| move | `async (path, { path, index }) -> await searchCompleted` | Moves the node at the provided path (first argument) to the target location provided in the second parameter. If the target path is not specified, it will default to the current node's group. If the target index isn't provided, it will default to moving to the end of the target group. You can await this for when updates settle and relevant searches are completed. Useful for drag and drop query builder interfaces that let you move nodes around in and between groups. |
-| pauseNested | `async (path) -> await searchCompleted` | Recursively set paused to true for the node at the path and all its children. You can await this for when updates settle and relevant searches are completed. |
-| unpauseNested | `async (path) -> await searchCompleted` | Recursively set paused to false for the node at the path and all its children. You can await this for when updates settle and relevant searches are completed. |
-| isPausedNested | `async (path) -> bool` | Returns a bool for whether the node at the path and all of its children are paused. |
+
+#### Actions
+
+Client methods that mutate or otherwise act on nodes are known as **actions**. Except for `isPausedNested`, you can await these for when updates settle and relevant searches are completed. Conventionally, actions take a node path as their first parameter. 
+
+Note that it's also possible to add custom actions to a client instance through the `addActions` method (see the table above for more details).
+
+| Name | Signature | Description |
+| ---- | --------- | ----------- |
+| add | `async (path, newNode, {index}) -> await searchCompleted` | Adds a node to the tree as a child of the specified path. You can optionally pass an options object as the third param to specify the index where to add the node. The node can be a group with children. |
+| remove | `async path -> await searchCompleted` | Removes a node at the specified path. Will also remove children of the target node. |
+| mutate | `async (path, deltas) -> await searchCompleted` | Mutates the node at the given path with the new values. |
+| triggerUpdate | `async () -> await searchCompleted` | Will trigger an update with a `none` reactor, updating only nodes that are already marked for update. This is useful when `disableAutoUpdate` is set to true. |
+| clear | `async path -> await searchCompleted` | Resets the node's values to those given on the node type's `defaults` (except `field`) |
+| replace | `async (path, newNode | transform) -> await searchCompleted` | Replaces the node at the given path with a new node. Accepts either a new node, or a `node -> newNode` transform function. If a function is given, it is called on the node at `path`, and the result is then used in `replace`. |
```

Comment:
putting a `|` here in the node signature wrapped in spaces is confusing markdown to think it's a column boundary

Reply:
huh, so it is. It looked fine in my VSCode preview ☹️